helm_deploy/hmpps-contacts-ui/values.yaml

```diff
@@ -51,9 +51,9 @@ generic-service:
     elasticache-redis:
       REDIS_HOST: "primary_endpoint_address"
       REDIS_AUTH_TOKEN: "auth_token"
-    #sqs-hmpps-audit-secret:
-    #  AUDIT_SQS_QUEUE_URL: "sqs_queue_url"
-    #  AUDIT_SQS_QUEUE_NAME: "sqs_queue_name"
+    sqs-hmpps-audit-secret:
+      AUDIT_SQS_QUEUE_URL: "sqs_queue_url"
+      AUDIT_SQS_QUEUE_NAME: "sqs_queue_name"
```

Comment:
This will fail to start up on dev without first creating the secret `sqs-hmpps-audit-secret` in the namespace. This is the bit of cloud platform work required (mentioned earlier)

Reply:
raised a [PR](https://github.com/ministryofjustice/cloud-platform-environments/pull/25425) and merged it now, this is ready for merge